docs/migration-recipe.md

```diff
@@ -0,0 +1,106 @@
+Migrating to new host:
+----------------------
+
+### Prepare for migration:  
```

Comment:
I think we should specify or make clear that this should be done locally.

Perhaps the title of this section should be "Prepare salt configuration for migration".